Controller/ElasticsearchController.php

```diff
@@ -612,6 +615,37 @@ public function searchApiAction(Request $request)
         ]);
     }
 
+    /**
+     * @param Request $request
```

Comment:
Remove docblock info that is already type hinted in the function definition

Reply:
Removed as recommended

Reply:
Rollbacked